docs/Demo.md

```diff
@@ -1,447 +1,409 @@
-        sed -i.original "s/your_org/${GITHUB_ORG}/g" mongo-persistent.yaml
-        oc create -f mongo-persistent.yaml
-    
+## Configuring Pelorus
+Follow the steps below to configure and apply the instance exporters for Pelorus to measure a sample application named `todolist` which will be deployed in the next procedure.
 
-The todolist application and mongo database should now build and deploy into the mongo-persistent namespace.
+**Procedure**
+1. Make a copy of the [values.yaml file](https://GitHub.com/konveyor/pelorus/blob/master/charts/pelorus/values.yaml) in the Pelorus repo ([raw link for curl-ing](https://raw.GitHubusercontent.com/konveyor/pelorus/master/charts/pelorus/values.yaml)) and save it to /var/tmp/values.yaml.
+```
+cp charts/pelorus/values.yaml /var/tmp/
+```
+1. Update the **exporters** section of the /var/tmp/values.yaml to match the following example.
+```
+exporters:
+  instances:
+  - app_name: deploytime-exporter
+    exporter_type: deploytime
+    extraEnv:
+    - name: LOG_LEVEL
+      value: DEBUG
+    - name: NAMESPACES
+      value: mongo-persistent
+  - app_name: committime-exporter
+    exporter_type: committime
+    extraEnv:
+    - name: LOG_LEVEL
+      value: DEBUG
+    - name: NAMESPACES
+      value: mongo-persistent
+```
+>**Note:** Documentation regarding values.yaml can be found on [Pelorus readthedocs.](https://pelorus.readthedocs.io/en/latest/Configuration/)
 
-* Check the build
+1. Apply the updated values for Pelorus.
+```
+helm upgrade pelorus charts/pelorus --namespace pelorus --values /var/tmp/values.yaml
+```
+> **Note:** Allow time for the the committime exporter pod to be deployed.
 
-        oc get all -n mongo-persistent
+Pelorus should now be installed and configured to measure the todolist sample app. Deploy the sample application to view measurements from Pelorus's Grafana dashboard next.
 
->***Note:*** Please pause to allow the todolist pod to build and deploy.
+## Installing the todolist sample application
+Pelorus is installed and configured to measure the todolist sample app. Follow the steps below to deploy the sample application and view measurements from the Pelorus Grafana dashboard.
 
-*  Ensure that your github fork of mig-demo-apps is correctly set as the uri value in the todolist BuildConfig.
-    
-        - kind: BuildConfig
-          apiVersion: build.openshift.io/v1
-          metadata:
-            name: todolist
-        <snip>
-              source:
-                type: Git
-                git:
-                  uri: https://github.com/weshayutin/mig-demo-apps.git
-                  ref: master
-    
-## View the Pelorus measurements
+**Procedure**
+1. git clone the forked copy of konveyor/mig-demo-apps.
+```
+git clone https://GitHub.com/your_org/mig-demo-apps.git
+```
+1. Install the todolist-mongo-go sample application.
+```
+cd mig-demo-apps/apps/todolist-mongo-go
+export GitHub_ORG=<YOUR_REAL_GitHub_FORK_ORG>
```

Comment:
here actually capital GITHUB_ORG, because it's env var.

Reply:
Should it be "GITHUB_FORK" as well?

Reply:
env variables should be in all caps "GITHUB_ORG"